floris/floris_model.py

```diff
@@ -885,51 +886,64 @@ def get_turbine_TIs(self) -> NDArrayFloat:
 
     ### Methods for sampling and visualization
 
+    def set_for_viz(self, findex: int, solver_settings: dict) -> None:
```

Comment:
Could/should the findex default to 0?  Does the user have to supply a findex if n_findex=0?

Reply:
Should have read more before commenting, this seems to be default in the higher-level functions